src/pages/idle-users.tsx

```diff
@@ -16,7 +16,7 @@ const IdleUsers: FC = () => {
   } = useFetch(IDLE_USERS_URL);
 
   useEffect(() => {
-    if (response) {
+    if (response && response.allUserStatus) {
```

Comment:
Good candidate for [Optional Chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining)?

Reply:
done